.gitignore

```diff
@@ -1,11 +0,0 @@
-# generated files
-*.pyc
-*.log
-
-# Packaging
-*.egg
-*.egg-info
-dist
-build
-
-*.nc
```

Comment:
No more `.gitignore`. Just create one locally and add itself in the ignore list. This was hiding some files that I needed to check and commit before.